src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -1387,97 +1378,102 @@ oneofSubDisjunctBinder = intercalate "_or_" . fmap oneofSubBinder
 
 -- ** Helpers to wrap/unwrap types for protobuf (de-)serialization
 
-wrapE
-    :: MonadError CompileError m
-    => TypeContext
-    -> DotProtoType
-    -> [DotProtoOption]
-    -> HsExp
-    -> m HsExp
-wrapE ctxt dpt opts e = case dpt of
+wrapE :: TypeContext -> DotProtoType -> [DotProtoOption] -> HsExp -> HsExp
```

Comment:
Most of the changes in this file are just style/formatting changes. This block is where the action is.